rasa/shared/core/domain.py

```diff
@@ -1424,8 +1431,8 @@ def cleaned_domain(self) -> Dict[Text, Any]:
         for slot in domain_data[KEY_SLOTS].values():
             if slot["initial_value"] is None:
                 del slot["initial_value"]
-            if slot["auto_fill"]:
-                del slot["auto_fill"]
+            # if slot["auto_fill"]:
+            del slot["auto_fill"]
```

Comment:
Now that `auto-fill` can only be `False` - I wasn't sure if to delete it or not? To make my life easy with test adaptation, I went for deletion, but not sure what would make more sense for Rasa-X.

Reply:
Probably makes sense to delete for now but longer run we probably will want to just drop the key entirely if it has no information content at all.

Reply:
Dropped the key entirely after we decided to remove the functionality entirely but warn about the removal.